go-client/generator/admin.csv

```diff
@@ -1,20 +1,20 @@
-RPC_CM_DROP_APP,DropApp,DropAppRequest,DropAppResponse
-RPC_CM_CREATE_APP,CreateApp,CreateAppRequest,CreateAppResponse
-RPC_CM_RECALL_APP,RecallApp,RecallAppRequest,RecallAppResponse
-RPC_CM_LIST_APPS,ListApps,ListAppsRequest,ListAppsResponse
+RPC_CM_DROP_APP,DropApp,ConfigurationDropAppRequest,ConfigurationDropAppResponse
```

Comment:
No compatiblity issue?

Reply:
There are some compatiblity issues, because the structure names changed.
go-client defined it's own idl before, they have the same structure content (the fields type and sequence number) but different names, the go-client side names are more simple and clean, but consider the consistency and maintainability, I think use the unify idl and break some compatiblity would be better, the go-client is not wildly used currently.

Reply:
Let's at least send a NOTICE email to the mailing list first...

Reply:
Will, after this patch merged.